landoapi/models/landing_job.py

```diff
@@ -282,12 +283,17 @@ def set_landed_reviewers(self, path: Path):
             approved_by = revision.data.get("approved_by")
             if not approved_by:
                 continue
-            data = {"peers_or_owners": []}
+
+            if "peers_and_owners" not in revision.data:
+                revision.data["peers_and_owners"] = []
+
             for reviewer in approved_by:
                 if reviewer in directory.peers_and_owners:
-                    data["peers_or_owners"].append(reviewer)
-            data.update(revision.data)
-            revision.data = data
+                    revision.data["peers_and_owners"].append(reviewer)
```

Comment:
True. Fixed, though this field is still not "perfect" since if there is a backout and a revision is landed again with a different approving reviewer than the original, both will be included in this field. I think that's OK for the scope/purpose of this field though.